api/client/container/unpause.go

```diff
@@ -0,0 +1,51 @@
+package container
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"github.com/docker/docker/api/client"
+	"github.com/docker/docker/cli"
+	"github.com/spf13/cobra"
+)
+
+type unpauseOptions struct {
+	containers []string
+}
+
+// NewUnpauseCommand creats a new cobra.Command for `docker unpause`
+func NewUnpauseCommand(dockerCli *client.DockerCli) *cobra.Command {
+	var opts unpauseOptions
+
+	cmd := &cobra.Command{
+		Use:   "unpause [OPTIONS] CONTAINER [CONTAINER...]",
```

Comment:
think we can remove the `[OPTIONS]` at some point here (because we only have `--help`), but we can do that for all commands that don't take options in another PR